src/settings/JobProfiles/ViewJobProfile/ViewJobProfile.js

```diff
@@ -97,10 +99,12 @@ const ViewJobProfileComponent = props => {
   const jobProfileData = () => {
     const jobProfile = resources.jobProfileView || {};
     const [record] = jobProfile.records || [];
+    const { failed } = jobProfile;
```

Comment:
The request can be failed because of server issues or invalid UUID, can we check for the response status (404 not found) instead?